molgenis-data/src/main/java/org/molgenis/data/RepositoryCollectionCapability.java

```diff
@@ -2,9 +2,23 @@
 
 public enum RepositoryCollectionCapability
 {
+	/**
+	 * Repository collections can be edited
+	 */
 	WRITABLE,
 
+	/**
+	 * Repository collections can be updated
+	 */
 	UPDATABLE,
 
-	META_DATA_PERSISTABLE
+	/**
+	 * Repository collections persist meta data
+	 */
+	META_DATA_PERSISTABLE,
+
+	/**
+	 * Repository can store data in an L1, L2 or L3 cache
+	 */
+	CACHEABLE
```

Comment:
Property is now defined for both Repository and Repository Collection, is this required?